src/content/changes/highlights/2023-05-18.md

```diff
@@ -0,0 +1,20 @@
+---
+date: 2023-05-18T14:00:00
+title: Highlights for the week ending May 18 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+- All management-cluster Flux installations (`flux-system`, `flux-giantswarm`) have been upgraded to Flux [v0.41.2](https://github.com/fluxcd/flux2/releases/tag/v0.41.2).
```

Comment:
From which version? any feature/bug to highlight?

Reply:
Updated to include key highlights